### ISSUE TITLE
remove Hidden Games from Active Servers list

### ISSUE THREAD

**Issue by EternalDawn1** [Facepunch/sbox-issues#5786]
### For?

S&Box

### What can't you do?

disables that the Server is Shown in Active servers

### How would you like it to work?

make a Boolean Property so it disables that the Server is Shown in Active servers

[Property] public bool ActiveServerShown { get ; set;} = true or false

### What have you tried?

...

**Comment by Retroeer** [comment]
This would only put a bandage on the bigger problem that is hidden games are not actually hidden, you can remove the game from the server list but anyone can just use the `lobbies` command and connect to your game, this problem is only worsened by the new `game` command that lets anyone play any game even if its hidden.
A current workaround is to just run `Game.Close` on clients that don't meet a certain requirement (like steamid).

**Comment by MD485** [comment]
I don't think it was really an active problem when people joined through the lobbies command.
When the UI itself is inviting you to click on incomplete projects it's a bit more prevelant an issue.

I understand the benefit of having the game modes easily accessible, "Hey guys I'm working on mario cart and want to test something, could you hop on quick?", and it saves the effort of explaining how to join using the lobbies command.

But in practicality I've seen people putting in effort making "For people who joined from the lobby browser" areas as opposed to just developing their game mode.

**Comment by trundlr** [comment]
A couple addendums if this gets looked at:

- The lobbies should still show up if you have permission to view the game (the same way games work in the menu). This will allow for dev teams to easily hop into each others lobbies when playtesting a hidden game.
- This is more of a backend permissions thing (and definitely out of scope), but it would be nice to be able to define a whitelist of people who have access to your game outside your organization. And then we can use the backend API to build grant/revoke stuff however we wish for playtests and whatnot.
